src/reporter.h

```diff
@@ -45,8 +45,7 @@ class MetadataReporter {
   void ReportMetadata();
 
   // Send the given set of metadata.
-  void SendMetadata(
-      std::map<std::string, MetadataStore::Metadata>&& metadata)
+  void SendMetadata(std::vector<MetadataStore::Metadata>&& metadata_list)
```

Comment:
I'd keep the name `metadata`...

Reply:
I'm less concerned about the implementation (some of the code uses the name `m` in places for the iterated value). To me, the name `metadata` makes sense because that's the set of metadata we're sending. Clients will look at the function declaration. However, I don't feel too strongly about this.

Reply:
Done